studentInfoOGB.py

```diff
@@ -0,0 +1,96 @@
+import random
+
+firstName = ["jay", "jim", "roy", "axel", "billy", "charlie", "jax", "gina", "paul",
+"ringo", "ally", "nicky", "cam", "ari", "trudie", "cal", "carl", "lady", "lauren",
+"ichabod", "arthur", "ashley", "drake", "kim", "julio", "lorraine", "floyd", "janet",
+"lydia", "charles", "pedro", "bradley"]
+
+lastName = ["blake", "holt", "meilijson", "brannon", "fieser", "line", "hellman",
+"lyndon", "wendell", "corr", "fricano", "carrigan", "giddings", "piesse", "charlwood",
+"thaler", "ambrose", "hamer", "schweig", "geiger", "kinder", "rogantin", "chevallier"
+"wang", "tiedemann", "cohn", "digel", "reheem", "dzieciololowski", "callanan",
+"lie", "oden"]
+
+def main():
+  students = [
+    Student("Larsson", 37),
+    Student("BonJovi", 55),
+  ]
+
+  printHeader()
+  selection = int(getUserSelection())
+  if selection == 0:
+    printStudentsByAge(students)
+  elif selection == 1:
+    printStudentsByLName(students)
+  elif selection == 2:
+    printStudentsByFName(students)
+  else:
+    print ("SELECTION NOT RECOGNIZED")
+
+
+class Student:
+  def __init__(self, lastName, age):
+    self.lastName = random.choice (lastName)
+    self.age = self.assignRandomAge()
```

Comment:
`self.age` is always `None`, because it is being set to the return value of `self.assignRandomAge()`, which is always `None`. When a function does not return anything, its return value is `None`.

`self.assignRandomAge()` doesn't return anything, but instead sets the value of `self.age` to a random int in the range of 0 to 100. If you want to randomly assign the age of the student, then calling the method would be sufficient (e.g., `self.assignRandomAge()` instead of `self.age = assignRandomAge()`)